client/client.go

```diff
@@ -39,44 +39,26 @@ func makeClient(cfg clientConfig) (Client, error) {
 	if !cfg.insecure && cfg.chainHash == nil && cfg.chainInfo == nil {
 		return nil, errors.New("No root of trust specified")
 	}
-	if len(cfg.urls) == 0 {
+	if len(cfg.clients) == 0 && cfg.watcher == nil {
```

Comment:
Are you using the condition "no clients but there is a watcher" later on ? I dont see it. If not, not sure this extra condition is useful ?

Reply:
If there are no clients, it will create an [EmptyClient](https://github.com/drand/drand/pull/511/files#diff-1cd41a2611da49ce63630d5a51bd82c5R61).
This allows for the "watcher with info" case we claim to support :)